src/layouts/homepage.html.jade

```diff
@@ -34,19 +34,20 @@ html(lang="en")
                         span.title Cozy&nbsp;
                         | a Personal Cloud
                         br
-                        | You can Host, Hack
-                        | and Delete
-                    h4
-                        | Your contacts, calendars and files are stored on your
-                        | hardware and synced with your devices
+                        | you can host, customize
+                        | and fully control
```

Comment:
As said yesterday on IRC, while "hack" has a negative connotation, customize is not the best replacement. +1 with @frankrousseau